packages/gasket-plugin-https-proxy/test/actions.test.js

```diff
@@ -0,0 +1,81 @@
+import { vi, describe, it, expect, beforeEach, afterEach } from 'vitest';
```

Comment:
Sweet. I'm also a big fan of `vitest`; it finally made modules just work.

Reply:
👍 becoming a fan also